enterprise/app/controllers/enterprise/api/v1/accounts_controller.rb

```diff
@@ -55,6 +55,24 @@ def toggle_deletion
     end
   end
 
+  def topup_options
```

Comment:
Let's keep the topup options in the frontend only, no need for a network call also. We can add the catalog later. On the backend we can validate from a hard coded array. Some duplication is fine here. Easier to write specs too, and the frontend will be simpler as well